manifests/process_exporter.pp

```diff
@@ -82,12 +83,18 @@
     default => undef,
   }
 
+  if $hash_watched_processes.empty() {
+    $config_path_content = template('prometheus/process-exporter.yaml.erb')
+  } else {
+    $config_path_content = $hash_watched_processes.to_yaml
```

Comment:
will this add the preceding `---`?

Reply:
yes it will and AFAIK it's not really an issue